meilisearch-release-changelog/v1.11.0.md

```diff
@@ -0,0 +1,257 @@
+
+```json5
+POST /multi-search
+{
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	"facetsByIndex": {
+	  "movies": ["title", "id"],
+	  "comics": ["title"],
+	}
+  },
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
+
+The multi-search response will include a new field, `facetsByIndex` with facet data separated per index:
+
+```json5
+{
+  "hits": […],
+  …
+  "facetsByIndex": {
+      "movies": {
+        "distribution": {
+          "title": {
+            "Batman returns": 1
+          },
+          "id": {
+            "42": 1
+          }
+        },
+        "stats": {
+          "id": {
+            "min": 42,
+            "max": 42
+          }
+        }
+      },
+     …
+  }
+}
+```
+
+#### Merged facet information
+
+To obtain facet distribution and stats for all index merged into a single ,use both `facetsByIndex` and `mergeFacets` when querying the `POST` `/multi-search` endpoint:
+
+```json5
+POST /multi-search
+{
+
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	  "facetsByIndex": {
+	    "movies": ["title", "id"],
+	    "comics": ["title"],
+	  },
+	  "mergeFacets": {
+	    "maxValuesPerFacet": 10,
+	  }
+  }
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
+
+The response includes two new fields, `facetDistribution` and `facetStarts`:
+
+```json5
+{
+  "hits": […],
+  …
+  "facetDistribution": {
+    "title": {
+      "Batman returns": 1
+      "Batman: the killing joke":
+    },
+    "id": {
+      "42": 1
+    }
+  },
+  "facetStats": {
+    "id": {
+      "min": 42,
+      "max": 42
+    }
+  }
+}
+```
+
+Done by @dureuill in #4929.
+
+## Experimental — New `STARTS_WITH` filter operator
+
+Enable the experimental feature to use the `STARTS_WITH` filter operator:
+
+```bash
+curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "containsFilter": true
+  }'
+```
+
+Use the `STARTS_WITH` operator when filtering:
+
+```json5
+curl \
+  -X POST http://localhost:7700/indexes/movies/search \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "filter": "hero STARTS_WITH spider"
+  }'
+```
+
+🗣️ This is an experimental feature, and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/763).
+
+Done by @Kerollmops in #4939.
+
+## Other improvements
+
+* Language support and [localizedAttributes settings](https://www.meilisearch.com/docs/reference/api/settings#localized-attributes) by @ManyTheFish in #4937
+  * Add ISO-639-1 variants
+  * Convert ISO-639-1 into ISO-639-3
+* Add a German language tokenizer by @luflow in meilisearch/charabia#303 and in #4945
```

Comment:
Changes related to [v1.11.0-rc.1 🐿️](https://github.com/meilisearch/meilisearch/releases/tag/untagged-3d1d8480461ba89f276c)
```suggestion
* Add a German language tokenizer by @luflow in meilisearch/charabia#303 and in #4945
* Improve Turkish language support by @tkhshtsh0917 in https://github.com/meilisearch/charabia/pull/305 and in  #4957
```